src/data/metadata.csv

```diff
@@ -8887,15 +8888,15 @@ GB_SCT_S08000029,GB,United Kingdom,SCT,Scotland,S08000029,Fife,,,,
 GB_SCT_S08000030,GB,United Kingdom,SCT,Scotland,S08000030,Tayside,,,,
 GB_SCT_S08000031,GB,United Kingdom,SCT,Scotland,S08000031,Greater Glasgow and Clyde,,,Greater Glasgow & Clyde,
 GB_SCT_S08000032,GB,United Kingdom,SCT,Scotland,S08000032,Lanarkshire,,,,
-GB_UKC,GB,United Kingdom,UKC,North East,,,,,E12000001,
-GB_UKD,GB,United Kingdom,UKD,North West,,,,,E12000002,
-GB_UKE,GB,United Kingdom,UKE,Yorkshire and the Humber,,,,,E12000003,
-GB_UKF,GB,United Kingdom,UKF,East Midlands,,,,,E12000004,
-GB_UKG,GB,United Kingdom,UKG,West Midlands,,,,,E12000005,
-GB_UKH,GB,United Kingdom,UKH,East of England,,,,,E12000006,
-GB_UKI,GB,United Kingdom,UKI,Greater London,,,,,E12000007,
-GB_UKJ,GB,United Kingdom,UKJ,South East,,,,,E12000008,
-GB_UKK,GB,United Kingdom,UKK,South West,,,,,E12000009,
+GB_UKC,GB,United Kingdom,UKC,North East England,,,,,E12000001,
+GB_UKD,GB,United Kingdom,UKD,North West England,,,,,E12000002,
+GB_UKE,GB,United Kingdom,UKE,"Yorkshire and the Humber, England",,,,,E12000003,
+GB_UKF,GB,United Kingdom,UKF,East Midlands England,,,,,E12000004,
+GB_UKG,GB,United Kingdom,UKG,West Midlands England,,,,,E12000005,
+GB_UKH,GB,United Kingdom,UKH,East of England England,,,,,E12000006,
```

Comment:
Not sure "East of England England" works, perhaps there is no need for the additional England here (or maybe format similar to Yorkshire and the Humber.

Reply:
Good catch, I'll remove the superfluous name